[32mINFO    [0m mylogger:logger.py:83 conftest.py::def setup_teardown(...)::line 52 > DELETE > 200 > {"message":"OK"}

[32mINFO    [0m mylogger:logger.py:83 conftest.py::def setup_teardown(...)::line 53 > DELETE > 200 > {"message":"OK"}

[32mINFO    [0m mylogger:logger.py:83 <setup> Cleared all users from DB
[32mINFO    [0m mylogger:logger.py:83 endpoints.py::def add_user(...)::line 70 > POST > 200 > {"data":"Knejwc","message":"OK"}
[32mINFO    [0m mylogger:logger.py:83 endpoints.py::def add_friend(...)::line 110 > PUT > 200 > {"data":"Bogsok","message":"OK"}

[32mINFO    [0m mylogger:logger.py:83 endpoints.py::def add_friend(...)::line 110 > PUT > 200 > {"error":"Bogsok already a friend of Knejwc"}
[33mWARNING [0m mylogger:logger.py:77 (log error) you try to pass a not string arguments... just removed.
[32mINFO    [0m mylogger:logger.py:83 Session Closed